### PR TITLE
Add ordinary kriging means and vars to eeraster

### PR DESCRIPTION
**Pull request by erickzul**

### CODE REVIEW

File: eeraster.py
Comment (rothn) [inline eeraster.py:290]: This is declared as a global but appears to be missing from the globals at the top of the file.

File: eeraster.py
Comment (rothn) [inline eeraster.py:302]: This is declared as a global but appears to be missing from the globals at the top of the file.

File: eeraster.py
Comment (rothn) [inline eeraster.py:305]: Do we want to explicitly reference the UAT instance here, even for production users?

File: eeraster.py
Comment (rothn) [inline eeraster.py:293]: Do we want to explicitly reference the UAT instance here, even for production users?

**Review by rothn — CHANGES_REQUESTED** [review]
Looks good in principle. Minor changes requested. Please also get @benwulfe 's approval before merging due to the sensitive timing.

**Comment by erickzul** [comment]
Made changes based on comments. PTAL.

**Comment by rothn** [comment]
I think maybe we missed a comment-- "Do we want to explicitly reference the UAT instance here, even for production users?"

**Comment by erickzul** [comment]
Used eeedf.project_name() to parameterize the project name.

**Comment by rothn** [comment]
Approved! Please make sure to get @benwulfe 's OK first as this is merging into ddf_common during a period of transition.

**Review by rothn — APPROVED** [review]

**Review by benwulfe — CHANGES_REQUESTED** [review]
This looks good, but I suspect this only will work for users that are site admins that have full gcp access.  Try uploading to production timberid-prd - you probably wont be able to see it until you are added as a site admin.  We need to have this running on timberid-prd because we need to use real timberid data -- not from test.

This means only site admins can do research which is probably ok for now -- but should go away once we move to bonafide GEE rasters.

just ACK this and I will approve.

**Review by benwulfe — APPROVED** [review]

**Pull request merged**
